CIP-1775/README.md

```diff
@@ -0,0 +1,160 @@
+---
+CIP: 1775
+Title: Delegated Spending Authorizations
+Authors: Ivan Mikushin <sorrel_cantos.0@icloud.com>
+Comments-URI:
+Status: Proposed
+Type: Standards Track
+Created: 2022-11-17
+License: CC-BY-4.0
+---
+
+# Delegated Spending Authorizations
+
+"Give me custody, or give me death!" - [@ArmySpies](https://twitter.com/ArmySpies/status/1592586212816355328)
+
+## Abstract
+
+We propose to allow users to delegate authorized spending of outputs in their wallets to smart contracts or other users,
+without sending said outputs out of their wallets.
+
+## Motivation
```

Comment:
I think this has an almost total overlap with https://github.com/cardano-foundation/CIPs/pull/309 in motivation. The main difference is that instead of using native scripts and proxy contracts to define the delegated authorization, this CIP instead proposes inlining this system directly into the UTXO definition

Reply:
@SebastienGllmt The outcome seems to be similar, but **only if** the datum (or `()` if not present in the output) could be passed to the script referenced by the `RequireScript` clause. This would indeed make CIP-38 equivalent: for every user of a DApp, a proxy script could be constructed which would enable both the user and the DApp to spend the output.

Wallets would then find such outputs and show as still being in their custody, but still manageable by other entities.

I do see a couple issues to iron out before I could get behind CIP-38:
1. We should specify what the `RequireScript` script is passed as its arguments. If it's all the same arguments as a regular Plutus validation script, this works for me. This also means, datum should be present in the output or implied as `()`.
2. Looks like this can only work starting with Plutus v3: we should make the Plutus script aware (via script context) of its non-exclusive control of the output.